.github/workflows/starter.yml

```diff
@@ -0,0 +1,15 @@
+name: Workflow starter
+
+on: [push, pull_request]
+
+jobs:
+    CodeQL:
+        uses: oneapi-src/unified-memory-framework/.github/workflows/codeql.yml@main
```

Comment:
since you're using the full path of these workflows - can you move it into a sub-dir, e.g. `workflows/checks/...yml`? some time ago it was not enabled in GHA, but now it seems it could work and it will lead to a bit better order in workflows

Reply:
It would be another PR.